### PR TITLE
Added the torch.polar function

### PR DESCRIPTION
**Pull request by Bloemenstraat**

### CODE REVIEW

**Comment by paulaehab** [comment]
Close #15480

**Comment by paulaehab** [comment]
Hey @Bloemenstraat  🙂, I will link this to the corresponding issue, but in future you should do this yourself by adding a comment ‘Close #Issue_number’, as explained here: (https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues)

File: .devcontainer/image/devcontainer.json [hunk 1]
```diff
@@ -41,4 +42,4 @@
 			"dockerDashComposeVersion": "v2"
 		}
     }
-}
```
Comment (paulaehab) [inline .devcontainer/image/devcontainer.json:44]: You should not make any change in any files except the files of the function so please remove this

File: ivy/functional/frontends/torch/creation_ops.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # local
 import ivy
+from torch import polar as torch_polar
```
[the remark this answers is not in the document]
Comment (Bloemenstraat) [reply ivy/functional/frontends/torch/creation_ops.py:3]: The problem stems from Ivy not containing a Complex function. There is no problem calculating the real and imaginary part using only Ivy function, but I can't seem to find a way to convert them to a complex array. Also, to my knowledge, there are no setters to real and img attributes of a complex array.

**Pull request closed without merging**